src/components/navbar.jsx

```diff
@@ -0,0 +1,14 @@
+const React = require('react');
+const Link = require('react-router').Link;
+
+const NavBar = React.createClass({
+  render: function() {
+    return (
+      
+    <p className="lead"><Link to='/' className="btn btn-info">Back to the Randoms</Link> | <Link to='/about' className="btn btn-primary">About the Randoms</Link></p>
```

Comment:
May benefit to separate these links onto separate lines. It'll be easier to read :)